diff --git a/live-build/auto/config b/live-build/auto/config
index 20d28139..8f188b5d 100755
--- a/live-build/auto/config
+++ b/live-build/auto/config
@@ -1048,12 +1048,13 @@ case $PROJECT in
 				add_task install ubuntu-server-raspi
 				;;
 			armhf*)
-				add_package install flash-kernel
+			  if [ "${SUBPROJECT:-}" != minimized ]; then
+				  add_package install flash-kernel
+				fi
 				;;
 			arm64*)
-				add_package install flash-kernel
-				if [ "${SUBARCH:-}" = "generic" ]; then
-					KERNEL_FLAVOURS=generic
+				if [ "${SUBPROJECT:-}" != minimized ]; then
+				  add_package install flash-kernel
 				fi
 				;;
 			amd64*)
diff --git a/live-build/ubuntu-cpc/hooks.d/base/disk-image-uefi.binary b/live-build/ubuntu-cpc/hooks.d/base/disk-image-uefi.binary
index 3685e464..4602267c 100755
--- a/live-build/ubuntu-cpc/hooks.d/base/disk-image-uefi.binary
+++ b/live-build/ubuntu-cpc/hooks.d/base/disk-image-uefi.binary
@@ -112,11 +112,11 @@ install_grub() {
     # please file a bug against grub2 to include the affected module.
     case $ARCH in
         arm64)
-            chroot mountpoint apt-get -qqy install --no-install-recommends shim-signed grub-efi-arm64-signed
+            chroot mountpoint apt-get install -qqy shim-signed
             efi_target=arm64-efi
             ;;
         armhf)
-            chroot mountpoint apt-get -qqy install --no-install-recommends grub-efi-arm grub-efi-arm-bin
+            chroot mountpoint apt-get -qqy install grub-efi-arm grub-efi-arm-bin
             efi_target=arm-efi
             ;;
         amd64)
