=== modified file 'BuildLiveCD'
--- BuildLiveCD	2007-08-03 14:10:49 +0000
+++ BuildLiveCD	2008-02-13 14:26:41 +0000
@@ -55,9 +55,9 @@
     echo "No valid suites to build for" >&2 && exit 1
 fi
 if (( $# == 0 )) || [ "X$1" = "Xall" ]; then
-    set -- ubuntu kubuntu edubuntu xubuntu base
+    set -- ubuntu kubuntu kubuntu-kde4 edubuntu xubuntu base
     if [ "$ARCH" = "i386" ]; then
-        set -- ubuntu ubuntu-lpia kubuntu edubuntu xubuntu base
+        set -- ubuntu ubuntu-lpia kubuntu kubuntu-kde4 edubuntu xubuntu base
     fi
 fi
 

=== modified file 'livecd.sh'
--- livecd.sh	2008-01-31 17:22:01 +0000
+++ livecd.sh	2008-02-13 14:30:01 +0000
@@ -129,15 +129,15 @@
 shift $((OPTIND-1))
 
 if (( $# == 0 )) || [ "X$1" = "Xall" ]; then
-    set -- ubuntu kubuntu edubuntu xubuntu gobuntu base
+    set -- ubuntu kubuntu kubuntu-kde4 edubuntu xubuntu gobuntu base
     if [ "$ARCH" = "i386" ]; then
-        set -- ubuntu ubuntu-lpia kubuntu edubuntu xubuntu gobuntu base
+        set -- ubuntu ubuntu-lpia kubuntu kubuntu-kde4 edubuntu xubuntu gobuntu base
     fi
 fi
 
 for arg in "$@"; do
     case "$arg" in
-	ubuntu|ubuntu-lpia|edubuntu|kubuntu|xubuntu|gobuntu|base|tocd)
+	ubuntu|ubuntu-lpia|edubuntu|kubuntu|kubuntu-kde4|xubuntu|gobuntu|base|tocd)
 	    ;;
 	*)
 	    echo bad name >&2;
@@ -177,6 +177,10 @@
 	    LIST="$LIST minimal^ standard^ kubuntu-desktop^"
 	    LIVELIST="kubuntu-live^ xresprobe laptop-detect casper lupin-casper"
 	    ;;
+	kubuntu-kde4)
+	    LIST="$LIST minimal^ standard^ kubuntu-kde4-desktop^"
+	    LIVELIST="kubuntu-kde4-live^ xresprobe laptop-detect casper lupin-casper"
+	    ;;
 	edubuntu)
 	    LIST="$LIST minimal^ standard^ edubuntu-desktop^"
 	    LIVELIST="edubuntu-live^ xresprobe laptop-detect casper lupin-casper"

